Crashes because No NameTypeIndex match for LONG_STANDARD

Hello, I had implemented fingerprintjs-android `2.0.0` and got crashes about TimeZoneNames with following details:

```txt
Fatal Exception: java.lang.AssertionError: No NameTypeIndex match for LONG_STANDARD
       at android.icu.impl.TimeZoneNamesImpl$ZNames.getNameTypeIndex(TimeZoneNamesImpl.java:724)
       at android.icu.impl.TimeZoneNamesImpl$ZNames.getName(TimeZoneNamesImpl.java:790)
       at android.icu.impl.TimeZoneNamesImpl.getTimeZoneDisplayName(TimeZoneNamesImpl.java:183)
       at android.icu.text.TimeZoneNames.getDisplayName(TimeZoneNames.java:261)
       at java.util.TimeZone.getDisplayName(TimeZone.java:405)
       at java.util.TimeZone.getDisplayName(TimeZone.java:322)
       at com.fingerprintjs.android.fingerprint.info_providers.DevicePersonalizationInfoProviderImpl$timezone$1.invoke(DevicePersonalizationInfoProvider.kt:46)
       at com.fingerprintjs.android.fingerprint.info_providers.DevicePersonalizationInfoProviderImpl$timezone$1.invoke(DevicePersonalizationInfoProvider.kt:46)
       at com.fingerprintjs.android.fingerprint.tools.ExceptionSafeExecutorKt.executeSafe(ExceptionSafeExecutor.kt:9)
       at com.fingerprintjs.android.fingerprint.info_providers.DevicePersonalizationInfoProviderImpl.timezone(DevicePersonalizationInfoProvider.kt:46)
       at com.fingerprintjs.android.fingerprint.fingerprinting_signals.FingerprintingSignalsProvider$timezoneSignal$2.invoke(FingerprintingSignalsProvider.kt:322)
       at com.fingerprintjs.android.fingerprint.fingerprinting_signals.FingerprintingSignalsProvider$timezoneSignal$2.invoke(FingerprintingSignalsProvider.kt:321)
       at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
       at com.fingerprintjs.android.fingerprint.fingerprinting_signals.FingerprintingSignalsProvider.getTimezoneSignal(FingerprintingSignalsProvider.kt:321)
       at com.fingerprintjs.android.fingerprint.tools.FingerprintingLegacySchemeSupportExtensions$getDeviceStateSignals$43.invoke(FingerprintingLegacySchemeSupportExtensions.kt:177)
       at com.fingerprintjs.android.fingerprint.tools.FingerprintingLegacySchemeSupportExtensions$getDeviceStateSignals$43.invoke(FingerprintingLegacySchemeSupportExtensions.kt:177)
       at com.fingerprintjs.android.fingerprint.tools.FingerprintingLegacySchemeSupportExtensions.createSignalIfNeeded(FingerprintingLegacySchemeSupportExtensions.kt:20)
       at com.fingerprintjs.android.fingerprint.tools.FingerprintingLegacySchemeSupportExtensions.getDeviceStateSignals(FingerprintingLegacySchemeSupportExtensions.kt:182)
       at com.fingerprintjs.android.fingerprint.Fingerprinter.getFingerprint$lambda-7(Fingerprinter.kt:188)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
       at java.lang.Thread.run(Thread.java:764)
```

Most of them happened on the **Android 8**. Any insight about this?

<img width="424" alt="Screen Shot 2023-06-21 at 13 52 35" src="https://github.com/fingerprintjs/fingerprintjs-android/assets/866289/d9aa7198-87b6-4dd6-abac-852ac0bb3825">

Thank you

--- a/fingerprint/lint.xml
+++ b/fingerprint/lint.xml
@@ -1,0 +1,4 @@
+<lint>
+    <!--  TODO: remove this rule and update to the latest target SDK -->
+    <issue id="OldTargetApi" severity="informational" />
+</lint>

--- a/fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/ExceptionSafeExecutor.kt
+++ b/fingerprint/src/main/java/com/fingerprintjs/android/fingerprint/tools/ExceptionSafeExecutor.kt
@@ -1,13 +1,10 @@
 package com.fingerprintjs.android.fingerprint.tools
 
-import java.lang.Exception
-
-
 @Deprecated(message = DeprecationMessages.UTIL_UNINTENDED_PUBLIC_API)
 public fun <T> executeSafe(code: () -> T, defaultValue: T): T {
     return try {
         code()
-    } catch (exception: Exception) {
+    } catch (_: Throwable) {
         defaultValue
     }
 }

